sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -283,6 +284,8 @@ def compute_table_name(row):
     'BigQuerySink',
     'WriteToBigQuery',
     'ReadFromBigQuery',
+    'ReadAllFromBigQueryRequest',
```

Comment:
Does the java api use a similar concept?

Reply:
I'm working on adding similar ReadAll to Java API.